backend/lib/piece-indexer.js

```diff
@@ -0,0 +1,49 @@
+import { fetchPayloadCid } from './pix-service/service.js'
+import { parseDeals, storeActiveDeals } from './deal-observer.js'
+
+/** @import {Queryable} from '@filecoin-station/deal-observer-db' */
+/** @import { Static } from '@sinclair/typebox' */
+/** @import { ActiveDealDbEntry } from '@filecoin-station/deal-observer-db/lib/types.js' */
+
+/**
+ * @param {import("@filecoin-station/deal-observer-db").Queryable} pgPool
+ * @param {function} makeRpcRequest
+ * @param {function} pixRequest
+ * @returns {Promise<void>}
+ */
+export async function updatePayloadCids(pgPool, makeRpcRequest, activeDeals, pixRequest) {
```

Comment:
```suggestion
export async function updatePayloadCids(pgPool, makeRpcRequest, activeDeals, makePixRequest) {
```

Name`pixRequest` does not make it obvious that it's a function that makes request to `pix` service.